tests/test_bounds.py

```diff
@@ -557,7 +515,7 @@ def foo(n: size, m: size, A: f32[n, m]):
     def bar(A: f32[10, 20]):
         foo(A)
 
-    assert bar.show_effects() == golden
+    assert str(bar) == golden
```

Comment:
This test doesn't test the partial eval at all. Shouldn't we just delete the golden for it?